### PR TITLE
Parameterize services

### PR DESCRIPTION
**Pull request by nikogura**
Preserved orion production service urls as defaults, but added ability to override all URLS via ENV vars.

Adds CircleCI config.

Removed Travis config.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -18,6 +18,28 @@ const WebSocket = require('ws');
 const utils = require('./utils');
 exports.utils = utils;
 
+// API URL
+global.orionApi = process.env.ORION_API_URL ? process.env.ORION_API_URL : "https://api.orionlabs.io" ;
```
Comment (tj10200) [inline src/main.js:22]: Should these default URL's be going through fastly? `observatory.orionlabs.io` vs `api.orionlabs.io`.  
On the same topic, should the defaults be prod? Or staging?

Comment (nikogura) [reply src/main.js:22]: I don't know the answer.  I was just trying to get the parameterization into the code.

As for defaults, I think they need to stay prod due to the fact that all current aster code doesn't expect to pass in any configuration.

File: src/utils.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ const callApi = (url, method = 'POST', status = 200, payload = {}) => {
  * @returns {Promise<Object>}
  */
 exports.lyre = (token, groups, message = '', media = '', target = '') => {
-  const url = process.env.LYRE_URL || 'https://lyre.api.orionaster.com/lyre';
+  const url = process.env.LYRE_URL || `${global.orionLyre}/lyre`;
```
Comment (amcnamara) [inline src/utils.js:52]: These url overrides conflict with your new approach, we should rip them out (including the `LOCRIS_OV2WAV` and `LOCRIS_STT` below).

File: src/main.js (hunk 1 above)
Comment (amcnamara) [reply src/main.js:22]: Ideally aster and these services should be in the same AZ (or kubernetes cluster), going out to fastly would be detrimental assuming the apis are performant enough... that being said they're _currently_ on opposite ends of the country.

**Review by amcnamara — APPROVED** [review]

File: test/main.test.js [hunk 3]
```diff
@@ -16,8 +16,8 @@ const OrionClientRewire = rewire('../src/main.js');
 describe('callOrion', () => {
   it('Should fail', () => {
     const callOrion = OrionClientRewire.__get__('callOrion');
-
-    return callOrion('123', 'http://mock.url').catch((reason) => {
+    // has to be a real url, else it times out waiting for a connection and the test fails- though not in the expected fashion.
```
Comment (amcnamara) [inline test/main.test.js:19]: This actually seems like a legitimately broken test, and should be fixed in the client code to work with `http://mock.url`... we should revert this change and leave this test broken.

Comment (nikogura) [reply test/main.test.js:19]: The test is that a bad token - in this case '123' should fail.  Sending it to a fake URL is testing something else.

Admittedly, none of this code should be hitting external URL's, and all of it should be mocked, but that's a larger effort.

Leaving a broken test would block merging.
